src/jit/importer.cpp

```diff
@@ -3208,7 +3201,7 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
         {
             NO_WAY("JIT must expand the intrinsic!");
         }
-        else if (IsIntrinsicImplementedByUserCall(intrinsicID))
+        else if ((retNode->gtFlags & GTF_CALL) != 0)
```

Comment:
This last if clause could use a comment. If I understand it correctly, you should say something like:
If we must expand the intrinsic, then retNode (the tree that corresponds to the intrinsic expansion) must be non-null, and the returned tree must not contain a call.